storybook/webpack.config.js

```diff
@@ -1,26 +1,13 @@
-// you can use this file to add your custom webpack plugins, loaders and anything you like.
-// This is just the basic way to add addional webpack configurations.
-// For more information refer the docs: https://goo.gl/qPbSyX
+// load the default config generator.
+const genDefaultConfig = require('@kadira/storybook/dist/server/config/defaults/webpack.config.js')
 
-// IMPORTANT
-// When you add this file, we won't add the default configurations which is similar
-// to "React Create App". This only has babel loader to load JavaScript.
+module.exports = function ModifyConfig(config, env) {
+  const newConfig = genDefaultConfig(config, env)
 
-module.exports = {
```

Comment:
running the storybook. A lot of `Module build failed: Unknown word` errors appear.

This is the code I use to add the loaders:
```
newConfig.module.loaders.push({
    test: /\.css$/,
    loader: 'style!css?localIdentName=[local]___[hash:base64:5]',
  })
  newConfig.module.loaders.push({
    test: /\.png$/,
    loader: 'url-loader',
    query: { mimetype: 'image/png' },
  })
```